.github/workflows/lint.yml

```diff
@@ -21,18 +21,22 @@ jobs:
 
   docs:
     name: Docs
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - name: Install Deps
+      - uses: actions/setup-python@v2
+      - name: Install terraform-docs
         run: |
-          # Temporary fix for https://github.com/actions/virtual-environments/issues/1811
-          brew untap local/homebrew-openssl
-          brew untap local/homebrew-python2
-          brew update
-          brew install pre-commit terraform-docs terraform
+          # Get the download url of the latest version of terraform-docs
+          tf_docs_download_url=$(curl -s https://api.github.com/repos/terraform-docs/terraform-docs/releases/latest | jq -rc '.assets[] | select( .name | contains("linux-amd64")).browser_download_url')
```

Comment:
📝 terraform-docs recommend way with latest version url

```
> curl -s https://api.github.com/repos/terraform-docs/terraform-docs/releases/latest | jq -rc '.assets[] | select( .name | contains("linux-amd64")).browser_download_url'
https://github.com/terraform-docs/terraform-docs/releases/download/v0.10.1/terraform-docs-v0.10.1-linux-amd64
```

![image](https://user-images.githubusercontent.com/915731/97509022-70b31100-19c4-11eb-960c-3df03758d719.png)

Reply:
by using the github api we'll always use the latest stable version of terraform-docs.